ompi/mpi/fortran/mpif-h/Makefile.am

```diff
@@ -28,15 +28,19 @@ include $(top_srcdir)/Makefile.ompi-rules
 SUBDIRS = profile
 
 #
-# OMPI_PRPOFILING_DEFINES flag is enabled when we want our MPI_* symbols
+# OMPI_PROFILING_DEFINES flag is enabled when we want our MPI_* symbols
 # to be replaced by PMPI_*. In other words, this flag decides
 # whether "profile/defines.h" is included or not. "profile/defines.h"
 # replaces all MPI_* symbols with PMPI_* symbols. In this directory,
-# we need it to be 0
+# we need it to be 1 when configure'd with --enable-mpi-profile
 #
 
 AM_CPPFLAGS = -DOMPI_PROFILE_LAYER=0 -DOMPI_COMPILING_FORTRAN_WRAPPERS=1
 
+if WANT_MPI_PROFILING
+AM_CPPFLAGS += -DOMPI_PROFILING_DEFINES=1
```

Comment:
I think you'll need the "else" clause here, too, right?  I.e., since you use `if OMPI_PROFILING_DEFINES`, you need to set `-DOMPI_PROFILING_DEFINES=0` in the false case.